SeamCarver.js

```diff
@@ -1,20 +1,52 @@
 "use strict";
 
+const RED = 0;
+const GREEN = 1;
+const BLUE = 2;
+const BORDER_ENERGY = 1000;
+
+/** Seam carver removes low energy seams in an image from HTML5 canvas. */
 class SeamCarver {
+
+    /**
+     *
+     * Init seam carver
+     *
+     * @param {HMLT5 canvas} canvas canvas with image on it.
+     *
+     */
     constructor(canvas) {
         this.width = canvas.width;
         this.height = canvas.height;
         this.picture = [];
```

Comment:
I think this could be made more efficient if we use `Int16Array` or maybe even `Int8Array` if we make a special case for border pixels.